src/Render.cc

```diff
@@ -57,13 +64,26 @@ namespace drafter {
             case JSONRenderFormat:
             {
                 refract::RenderJSONVisitor renderer;
-                refract::IElement* element = MSONToRefract(attributes);
+
+                refract::IElement* element = MSONToRefract(*attributes);
+
+                if (!element) {
+                    return body;
+                }
+
```

Comment:
Modify element is OK, because you "say it" by non-const pointer, If you wan to say, "don't modify this parameter" you will use const pointer/reference/value.